tools/probe-ctrl/cmd/probe-ctrl/main.go

```diff
@@ -0,0 +1,79 @@
+package main
+
+import (
+	"flag"
+	"fmt"
+	"os"
+	"time"
+
+	"github.com/cilium/ebpf/link"
+)
+
+var (
+	argExec   string
+	argSymbol string
+	argClear  bool
+)
+
+var (
+	pinPath = "/sys/fs/bpf/strobelight-ctrl/"
```

Comment:
Does this mean it controls strobelight?
So completely unrelated to this project?
Could there be a comment or readme explaining the dependencies or how to use this?

Reply:
> Does this mean it controls strobelight?

No - `probe-ctrl` is just inspired by the blog post [1](https://engineering.fb.com/2025/01/21/production-engineering/strobelight-a-profiling-service-built-on-open-source-technology/). While the lifetime of probe-ctrl is short, the pinPath makes sure the probe stays attached exceeding the lifetime of probe-ctrl. When renaming this tool, I just forgot to adapt this path and so strobelight was still in it.

> Could there be a comment or readme explaining the dependencies or how to use this?

I have added a quick README with https://github.com/open-telemetry/opentelemetry-ebpf-profiler/pull/662/commits/92d8bed1ff1e34da07d94563dcef06190cc2eb4b.

Reply:
The documented profiler flag does not exist in `main` branch. It seems it is added in PR #651 so it does not make sense to merge this before the other PR is merged?

Reply:
yeah - i'm hoping to get some feedback on https://github.com/open-telemetry/opentelemetry-ebpf-profiler/pull/651